firefox-ios/Client/Application/WindowManager+DebugUtilities.swift

```diff
@@ -15,8 +15,10 @@ extension WindowManagerImplementation {
         var result = "----------- Window Debug Info ------------\n"
         result.append("Open windows (\(windows.count)) & normal tabs (via TabManager):\n")
         for (idx, (uuid, _)) in windows.enumerated() {
-            result.append("    \(idx + 1): \(short(uuid))\n")
             let tabMgr = tabManager(for: uuid)
+            let window = windows[uuid]?.sceneCoordinator?.window
```

Comment:
(Note: changes in this file are not related to fix, this is an internal dev utility for debugging.)